src/app/models/types/translations.ts

```diff
@@ -9,6 +9,7 @@ export interface Translations {
   ads: {
     advertisementLabel: string;
   };
+  recommendationTitle?: string;
```

Comment:
This can also be removed from here & the config file as the functionaity has been deleted